app/views/catalog/_search_results.html.erb

```diff
@@ -1,5 +1,5 @@
 <%# locals: (content_classes: 'col-md-8 col-xxl-6 px-xxl-4 ps-lg-4 ps-2', hide_constraints: false, hide_facets: false) %>
-<% if page_location.access_point? %>
+<% if page_location.access_point? || page_location.collection? %>
   <% if page_location.collection? && get_collection.present? %>
```

Comment:
Should this case move up to the top level (e.g. I don't think we want to nest this).